python/pyarrow/_s3fs.pyx

```diff
@@ -54,6 +54,31 @@ def finalize_s3():
     check_status(CFinalizeS3())
 
 
+def resolve_s3_region(bucket):
```

Comment:
Should we add this to `docs/api/filesystems.rst` ?

Reply:
Good point, I'll do it.